app/README.md

```diff
@@ -55,6 +54,12 @@ yarn proxy:start     - start the backend proxy and serve mock data
 yarn svelte:clean    - remove build artifacts
 yarn svelte:build    - compile svelte to JS
 yarn svelte:watch    - compile svelte to JS on every change to the code
+
+yarn generate:colors - update color CSS variables in public/colors.css from
+                       colors.json
```

Comment:
Should we explain the process for introducing/changing new colors in more detail in an extra paragraph?

Reply:
Good point, will do.